solutions/Making-choices-choices-with-functions-R.txt

```diff
@@ -0,0 +1,3 @@
+[1] "CM - 0.866"
```

Comment:
I like number 1. The challenge is that it can't be a vector since it's mixing numbers and strings, so it would have to be a list. I think this is OK if we give them a chunk of code in the exercise that helps them do this. We talk about lists the following week so this would be a preview.